CHANGES.rst

```diff
@@ -228,6 +228,14 @@ API Changes
   - Moved ``units.cgs.emu`` to ``units.deprecated.emu`` due to ambiguous
     definition of "emu". [#4918, #5906]
 
+  - ``jupiterMass``, ``earthMass``, ``jupiterRad``, and ``earthRad`` no longer
+    have their prefixed units included in the standard units.  If needed, they
+    can still  be found in ``units.deprecated``. [#5661]
+    
+  - ``solLum``,``solMass``, and ``solRad`` no longer have  their prefixed units
+    included in the standard units.  If needed, they can still be found in  
+    ``units.vounit_needed``, and are enabled by default. [#5661]
```

Comment:
I think the fully descriptive name would be `required_by_the_vounit_standard_but_not_actually_ever_used_by_an_astronomer_in_real_life_so_would_be_confusing_to_have_in_the_main_namespace`.  But that's 141 characters, which is > 80 :wink:.

Maybe `required_for_vounit`?  Or `prefixed_vounits`?